Components/BlockImageTextParallax/functions.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Flynt\Components\BlockImageTextParallax;
+
+use Flynt\Api;
+
+Api::registerFields('BlockImageTextParallax', [
+    'layout' => [
+        'name' => 'blockImageTextParallax',
+        'label' => 'Block: Image Text Parallax',
+        'sub_fields' => [
+            [
+                'label' => 'General',
+                'name' => 'generalTab',
+                'type' => 'tab',
+                'placement' => 'top',
+                'endpoint' => 0
+            ],
+            [
+                'label' => 'Image Position',
+                'name' => 'imagePosition',
+                'type' => 'button_group',
+                'choices' => [
+                    'imageLeft' => '<i class=\'dashicons dashicons-align-left\' title=\'Image on the left\'></i>',
+                    'imageRight' => '<i class=\'dashicons dashicons-align-right\' title=\'Image on the right\'></i>'
+                ]
+            ],
+            [
+                'label' => 'Image',
+                'name' => 'image',
+                'type' => 'image',
+                'preview_size' => 'medium',
+                'instructions' => '',
+                'max_size' => 4,
+                'required' => true,
+                'mime_types' => 'gif,jpg,jpeg,png'
```

Comment:
Considering performance, I would suggest to remove gif support here. It's also not 100% compatible with resizeDynamic as far as I know